connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TransformationChain.java

```diff
@@ -40,6 +44,8 @@ public R apply(R record) {
         for (final Transformation<R> transformation : transformations) {
             final R current = record;
 
+            log.trace("Applying SMT transformation {} to {}",
```

Comment:
I considered this "ATM Machine" effect when I wrote this, but decided that putting every name for something helps with readability more than grammar does. It also will help with grepping/searching the code base. If you feel strongly, I think a strong compromise would be "Applying Single Message Transformation (SMT)..."

Reply:
I don't see how wrong usage elsewhere inspires wrong usage here. 
Despite that, I still prefer `Applying transformation` since this log message is not a paragraph in a text where we need to be precise and need to list the acronym explicitly along with what it means. This is a log message that needs to be succinct and can be read in context. A transformation here implies a connect SMT.